### PR TITLE
Support NeuralNetwork.toFunction() when relu or leaky-relu is the activation function

### PR DESCRIPTION
**Pull request by jcjolley**
Without the IIFE, calling toFunction() on a neural network with relu, or leaky-relu set as the activation function causes toFunction() to throw a "SyntaxError: Unexpected token var"

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Author's Checklist:

- [x] My code focuses on the main motivation and avoids scope creep.
- [ ] My code passes current tests and adds new tests where possible.
- [ ] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [x] I have updated the documentation as needed.

## Reviewer's Checklist:
- [ ] I kept my comments to the author positive, specific, and productive.
- [ ] I tested the code and didn't find any new problems.
- [ ] I think the motivation is good for the project.
- [ ] I think the code works to satisfies the motivation.

### CODE REVIEW

File: src/neural-network.js [hunk 1]
```diff
@@ -885,9 +885,9 @@ export default class NeuralNetwork {
         case 'sigmoid':
           return `1/(1+1/Math.exp(${result.join('')}))`;
         case 'relu':
-          return `var sum = ${result.join('')};(sum < 0 ? 0 : sum);`;
+          return `(function () { var sum = ${result.join('')};(sum < 0 ? 0 : sum);})()`;
```
Comment (robertleeplummerjr) [inline src/neural-network.js]: This looks a little funny.  The functions don't return anything.

Comment (jcjolley) [reply src/neural-network.js]: Whoops.  Good call.

**Comment by robertleeplummerjr** [comment]
Adding a lambda in that location is going to be called a ridiculously ridiculous (see what I did there) amount of times.  I think it would be prudent to follow either the existing non-lambda means or putting a lambda in a reusable location.

**Review by jcjolley — COMMENTED** [review]
That look better?

**Comment by jcjolley** [comment]
@robertleeplummerjr Yeah, that will be called a ton of times.  The problem is, we can't declare 'var sum' inside of an array, which is where those end up, so we either need to find a way to inline the sum, or make a function out of it.

**Comment by robertleeplummerjr** [comment]
I'll look into this later today.

**Comment by robertleeplummerjr** [comment]
Resolved, but want to fix a couple issues before releasing: https://github.com/BrainJS/brain.js/pull/257

**Pull request closed without merging**